svgnative/include/svgnative/SVGDocument.h

```diff
@@ -174,6 +175,33 @@ class SVG_IMP_EXP SVGDocument
      */
     void Render(const char* id, const ColorMap& colorMap, float width, float height);
 
+    /**
+     * Retrieves the bounds of the SVG Document.
+     *
+     * The bounds are as tightly computed as possible. The bounds are based on the SVG canvas
+     * and don't take into account the viewBox on the root node. The rendering port's canvas
+     * is used to calculate the bounds so the user must make sure that a context has been
+     * set, otherwise an assertion will fire.
+     */
+    Rect Bounds();
```

Comment:
Should this be pure virtual? Is it needed to have SVGDocument::Bounds?

Reply:
Name suggestion GetBoundingBox(),

also, can it be const?

Reply:
By virtual you mean defined in header-only?

About the name, the rest of the functions such as Width and Height don't have a Get? So does it make sense to name it GetBoundingBox? It is for sure more meaningful so I don't have any problem changing it.

Reply:
Also, is an empty rectangle really indicative of a problem? One can have dumb paths like `<path d="M 100,100" />` or the standard doesn't allow that?